463.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def islandPerimeter(self, grid: List[List[int]]) -> int:
+        R, C = len(grid), len(grid[0])
+        perimeter = 0
+
+        for i in range(R):
+            for j in range(C):
+                if grid[i][j] == 1:
+                    if i - 1 < 0 or (i - 1 >= 0 and grid[i - 1][j]) == 0:
```

Comment:
There is no need to write `i - 1 >= 0` after `i - 1 < 0` due to logical operators' [short circuiting](https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not) behavior in Python.

Also, `i - 1 < 0` can be replaced by `i == 0` for better readability.